treeherder/etl/perf_data_adapters.py

```diff
@@ -235,6 +234,48 @@ def adapt_and_load(self, reference_data, job_data, datum):
             result_set_id = job_data[_job_guid]['result_set_id']
             push_timestamp = job_data[_job_guid]['push_timestamp']
 
+            # counters will not be part of the summary series
+            # counters have a json obj {'stat': val} instead of [val1, val2, ...]
+            for _test in talos_datum["talos_counters"].keys():
```

Comment:
my latest code should have check for that, maybe it didn't get pushed?  I will fix it with the other items.